hrstaffdir/tests/phpunit/bootstrap.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+ini_set('memory_limit', '2G');
+ini_set('safe_mode', 0);
+eval(cv('php:boot --level=classloader', 'phpcode'));
```

Comment:
I tried to run one of the tests in this extension and got this error:
![captura de tela 2016-06-30 as 08 31 29](https://cloud.githubusercontent.com/assets/388373/16486754/22df12d2-3e9d-11e6-8c4a-658e91896467.png)

To fix that, I had to change this line to:
`eval(cv('php:boot --level=full -t', 'phpcode'));`

After that I was able to run the tests, but they are failing. If you didn't try to run the WebTests yet, I think it would be a good idea to do it and check if there are more failing tests and create tickets to fix them later